docs/en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -248,6 +248,23 @@ Additionally, there are community supported guides for installing Silverstripe C
 * [Vagrant with silverstripe-australia/vagrant-environment](https://github.com/silverstripe-australia/vagrant-environment)
 * [Vagrant with BetterBrief/vagrant-skeleton](https://github.com/BetterBrief/vagrant-skeleton)
 
+### Email
+
+New installations of Silverstripe CMS are automatically configured to use a `sendmail` binary.  By default it will look in `/usr/sbin/sendmail` and is this configurable via `sendmail_path` in `php.ini`. Alternatively email can be configured to use SMTP or other mail transports instead of sendmail.
+
+You _must_ ensure emails are being sent from your _production_ environment. You can do this by testing that the
+"Lost password" form available at `/Security/lostpassword` sends an email to your inbox, or with the following code
+snippet:
+
+```
+$email = new Email('no-reply@mydomain.com', 'myuser@gmail.com', 'My test subject', 'My email body text');
```

Comment:
Probably should add `->send()` to this test snippet 😄

Reply:
Updated